pages_builder/generate_pages.py

```diff
@@ -107,14 +148,18 @@ def render_page(self, input_file):
                                 {{#examples}}
                                     {{#title}}<h2>{{title}}</h2>{{/title}}
                                     {{{markup}}}
-                                    <pre>{{markup}}</pre>
+                                    <div class="code open"><p class="code-label">Template ({{ templateFile }})</p>{{{highlighted_markup}}}</div>
```

Comment:
Looking at how they appear in the page, it would be good for these code labels to be `h2`s so the structure is:

h1: Name of the pattern
h2: Template
h2: Parameters